DigitalLearningSolutions.Web/appsettings.Development.json

```diff
@@ -19,5 +19,10 @@
     "PricingPage": true,
     "RefactoredFindYourCentrePage": true
   },
-  "LearningHubOpenAPIBaseUrl": "https://uks-learninghubnhsuk-openapi-test.azurewebsites.net"
+  "LearningHubOpenAPIBaseUrl": "https://uks-learninghubnhsuk-openapi-test.azurewebsites.net",
+  "LearningHubReportAPIConfig": {
+    "BaseUrl": "https://reportapi-test.test-learninghub.org.uk",
+    "ClientId": "BCDE972F-E007-4DFF-B233-8870474AEEAA",
+    "ClientIdentityKey": "56B1C9EF-9F25-4F8C-B771-85DCB2097B2E"

```

Comment:
@Swapnamol, is this a secret? Does it need moving to secrets.config?

Reply:
@kevwhitt-hee ClientIdentityKey moved to secrets.config.